gwsurrogate/catalog.py

```diff
@@ -109,6 +109,17 @@
   '''https://journals.aps.org/prd/abstract/10.1103/PhysRevD.99.064045''',
   'b42cd577f497b1db3da14f1e4ee0ccd1')
 
+_surrogate_world['NRHybSur2dq15'] = \
+  surrogate_info(\
+  'https://zenodo.org/record/3348115/files/NRHybSur2dq15.h5',
```

Comment:
404, not found!

Reply:
I currently have the .h5 files stored in my google drive. I was going to bring this up during the surrogate call tomorrow.